webapp/src/js/controllers/navigation-confirm.js

```diff
@@ -9,6 +10,9 @@ angular.module('inboxControllers').controller('NavigationConfirmCtrl',
 
     $scope.cancel = function() {
       $uibModalInstance.dismiss();
+      if ($state.current.name.includes('report')) {
+        $('div.material').removeClass('ng-hide');
```

Comment:
Definitely avoid mixing jquery and angular - they don't play nicely.

What you've done here is used jquery to remove the class that angular added based on the angular $state. This means the $state and the template are out of sync. This could be very bad.

What you need to do is modify the angular $state in such a way that angular removes the `ng-hide` class for you.

I'm not quite sure what issue you're trying to solve here - send me a DM next time we're both online together and I'll see if I can work out what's going on.